.github/actionlint.yml

```diff
@@ -4,10 +4,10 @@ self-hosted-runner:
     - dev
     - gen3
     - large
-    # Remove `macos-14` from the list after https://github.com/rhysd/actionlint/pull/392 is merged.
-    - macos-14
     - small
     - us-east-2
+    - ubuntu-22.04-arm64
+    - ubuntu-latest-arm64
```

Comment:
yes, that was the main idea - to use almost the same names so that you can easily move from GitHub-hosted to self-hosted runners. I mean just replace `runs-on: ubuntu-latest` with `runs-on: [self-hosted, ubuntu-latest]` to use our more powerful runners. Do you think this is a bad idea?

Reply:
I think it's better to give them more distinguishable label because GitHub's runners are ephemeral, and self-hosted runners are long-running, so we need to be more careful with self-hosted runners to not break them for the next jobs

Reply:
label changed to `large-arm64`